presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+=================
+Iceberg Connector
+=================
+
+Overview
+--------
+
+Apache Iceberg is an open table format for huge analytic datasets.
+The Iceberg connector allows querying data stored in
+files written in Iceberg format, as defined in the
+`Iceberg Table Spec <https://iceberg.apache.org/spec/>`_.
+
+The Iceberg connector supports:
+
+* All Presto types, including TIME, TIMESTAMP, DECIMAL
+* CREATE TABLE and CREATE TABLE AS
+* Partitioning tables by any Presto type
+* Dropping tables
+* Renaming tables
+* Deletion by partition
+* Table and column statistics
+* Table and column comments
+* Rolling back a table to a previous state
+* Schema evolution
+
+The Iceberg table state is maintained in metadata files. All changes to table state
+create a new metadata file and replace the old metadata with an atomic swap.
+The table metadata file tracks the table schema, partitioning config,
+custom properties, and snapshots of the table contents.
+
+Iceberg data files can be stored in either Parquet or ORC format, as
+determined by the ``format`` property in the table definition.  The
+table ``format`` defaults to ``ORC``.
+
+Iceberg is designed to improve on the known scalabilty limitations of Hive.  Hive
+stores table metadata in an object store backed by a database like MySQL.  It tracks
+partition locations in the metastore, but not individual data files.  Presto queries
+using the Hive connector must first call the metastore to get partition locations,
+then call the underlying filesystem to list all data files inside each partition,
+and then read metadata from each data file.
+
+Since Iceberg stores the paths to data files in the metadata files, it
+only consults the underlying file system for files that must be read.
+
+Status
+------
+
+The Iceberg connector is not yet supported for production use.
```

Comment:
Let's remove this now, as it should be usable, as shown by our tests

Reply:
Removed